gamemodes/terrortown/gamemode/server/sv_main.lua

```diff
@@ -204,6 +204,18 @@ local ttt_dbgwin = CreateConVar("ttt_debug_preventwin", "0", {FCVAR_NOTIFY, FCVA
 -- @realm server
 local ttt_newroles_enabled = CreateConVar("ttt_newroles_enabled", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE})
 
+---
+-- @realm server
+local ttt2_enable_thirdparty_workshop = CreateConVar("ttt2_thirdparty_enable_workshop", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE, FCVAR_REPLICATED})
+
+if ttt2_enable_thirdparty_workshop:GetBool() then
+	resource.AddWorkshop("1525218777") -- vFire
+end
+
+---
+-- @realm server
+CreateConVar("ttt2_thirdparty_enable_vfire", vFireInstalled and "1" or "0", {FCVAR_NOTIFY, FCVAR_ARCHIVE, FCVAR_REPLICATED})
```

Comment:
```suggestion
CreateConVar("ttt2_thirdparty_enable_vfire", "1", {FCVAR_NOTIFY, FCVAR_ARCHIVE, FCVAR_REPLICATED})
```

I don't see why you make that dependant on the fact if the addon is installed or not. You could gray out the checkbox if the addon is not installed